Sources/Turf/Geometries/Polygon.swift

```diff
@@ -14,6 +14,28 @@ public struct Polygon: Equatable {
     public init(outerRing: Ring, innerRings: [Ring] = []) {
         self.coordinates = ([outerRing] + innerRings).map { $0.coordinates }
     }
+
+    /**
+     Returns a circular polygon based on a given radius and a defined
+     number of vertices for precision.
+
+     - Parameter center: The center coordinate for the circle.
+     - Parameter radius: The radius of the circle, measured in meters.
+     - Parameter vertices: The number of vertices the circle will have. Defaults to 64.
+     - Returns: A polygon resembling a circle.
```

Comment:
It would be more accurate to describe this initializer as producing a regular polygon centered at the given coordinate. We can document it as a way to approximate a circle as well.